src/pocketmine/level/Explosion.php

```diff
@@ -209,7 +213,20 @@ public function explodeB() : bool{
 
 			$t = $this->level->getTile($block);
 			if($t instanceof Tile){
```

Comment:
Better not call `Level::getTile()` if `$yieldDrops` is not true
```php
if($yieldDrops){
    $t = $this->level->getTile($block);
    if($t instanceof Container){
        //...
    }
}
```

Reply:
@Muqsit and what about tiles that need to be removed which are not containers?

Reply:
updated.